_static/css/custom.css

```diff
@@ -1130,4 +1125,36 @@ div.topbar {
 
 .bd-topbar-whitespace {
     max-height:45px;
+}
+
+.custom-bdtoc .tocsection, .custom-bdtoc #bd-toc-nav {
+    border-left: 1px solid #fc7a4a !important;
+}
+
+.custom-bdtoc {
+    right: 35px;
+    padding-bottom: 10px;
+}
+
+.bd-search .icon {
+    top:27px;
+}
+
+input.rounded {
+    border-radius: 50%;
```

Comment:
border-radius: 5%

Reply:
great stuff!

Reply:
Done, was the wrong selector. Set to 10px.